classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(_label)
+    @label = label
+    @students = []
+  end
```

Comment:
- [ ] This typo as highlighted by the previous reviewer needs to be fixed. The constructor parameter is passed in as `_label` and called in a `label`. remove the underscore in the parameter 👍

Reply:
@Shedrack-Sunday 
thanks for your good and valuable feedback. 
I will take your comment in to account.